src/__template__/ComponentTemplate.vue

```diff
@@ -155,13 +156,15 @@ const svgProcessedContent = `${svgTitleContent}${!props.staticIds ? prefixSvgIds
     class="kui-icon {%%KONG_COMPONENT_ICON_CLASS%%}"
     data-testid="kui-icon-wrapper-{%%KONG_COMPONENT_ICON_CLASS%%}"
     :style="rootElementStyles"
+    :tabindex="decorative ? '-1' : undefined"
   >
     <svg
       :aria-hidden="decorative ? 'true' : undefined"
       data-testid="kui-icon-svg-{%%KONG_COMPONENT_ICON_CLASS%%}"
       fill="none"
       height="100%"
       role="img"
+      :tabindex="decorative ? '-1' : undefined"
```

Comment:
You shouldn't need the tabindex on both here; just on the parent should be sufficient.
```suggestion
    :tabindex="decorative ? '-1' : undefined"
  >
    <svg
      :aria-hidden="decorative ? 'true' : undefined"
      data-testid="kui-icon-svg-{%%KONG_COMPONENT_ICON_CLASS%%}"
      fill="none"
      height="100%"
      role="img"
```

Reply:
Done